src/resources/i18n/en.json

```diff
@@ -4,7 +4,10 @@
     "incorrectcredentials": "Invalid username or password",
     "passwordtooshort": "Password too short",
     "badrequest": "Bad request",
-    "internalerror": "An internal error occured"
+    "internalerror": "An internal error occured",
+    "lab": {
+      "someexamswithoutresultpleasecheck": "Some exams without result please check !"
```

Comment:
```suggestion
      "someexamswithoutresultpleasecheck": "Some exams without result please check."
```

Reply:
and probably in ExamForm we need `<InfoBox type="error" message={t(errorMessage)} />`

Reply:
There's no need of esclamation point, see my suggestions please